<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetadataImportSettingsDialog</class>
 <widget class="QDialog" name="MetadataImportSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Metadata Import Settings</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout_">
   <item>
    <widget class="QGroupBox" name="file_info_group_">
     <property name="title">
      <string>File Information</string>
     </property>
     <layout class="QFormLayout" name="file_info_layout_">
      <item row="0" column="0">
       <widget class="QLabel" name="file_path_label_">
        <property name="text">
         <string>File:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="file_path_value_">
        <property name="text">
         <string>/path/to/data.txt</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dimensions_label_">
        <property name="text">
         <string>Dimensions:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="dimensions_value_">
        <property name="text">
         <string>1024 x 768</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="range_group_">
     <property name="title">
      <string>Data Range Settings</string>
     </property>
     <layout class="QVBoxLayout" name="range_layout_">
      <item>
       <layout class="QFormLayout" name="range_form_layout_">
        <item row="0" column="0">
         <widget class="QLabel" name="range_min_label_">
          <property name="text">
           <string>Minimum value:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="range_min_spinbox_">
          <property name="minimum">
           <double>-999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="range_max_label_">
          <property name="text">
           <string>Maximum value:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="range_max_spinbox_">
          <property name="minimum">
           <double>-999999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>999999.000000000000000</double>
          </property>
          <property name="decimals">
           <number>6</number>
          </property>
          <property name="value">
           <double>100.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="out_of_range_label_">
        <property name="text">
         <string>Out-of-range handling:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="clamp_radio_">
        <property name="text">
         <string>Clamp to bounds</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="zero_radio_">
        <property name="text">
         <string>Set to zero (minimum)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="max_radio_">
        <property name="text">
         <string>Set to maximum</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="crop_group_">
     <property name="title">
      <string>Crop Region</string>
     </property>
     <layout class="QVBoxLayout" name="crop_layout_">
      <item>
       <widget class="QCheckBox" name="crop_enabled_checkbox_">
        <property name="text">
         <string>Enable cropping</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="crop_form_layout_">
        <item row="0" column="0">
         <widget class="QLabel" name="crop_start_label_">
          <property name="text">
           <string>Start (top-left):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="crop_start_layout_">
          <item>
           <widget class="QLabel" name="crop_start_x_label_">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="crop_start_x_spinbox_">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="crop_start_y_label_">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="crop_start_y_spinbox_">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="crop_end_label_">
          <property name="text">
           <string>End (bottom-right):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="crop_end_layout_">
          <item>
           <widget class="QLabel" name="crop_end_x_label_">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="crop_end_x_spinbox_">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="crop_end_y_label_">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="crop_end_y_spinbox_">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>99999</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="vertical_spacer_">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout_">
     <item>
      <spacer name="button_spacer_">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="import_button_">
       <property name="text">
        <string>Import</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button_">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>import_button_</sender>
   <signal>clicked()</signal>
   <receiver>MetadataImportSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_button_</sender>
   <signal>clicked()</signal>
   <receiver>MetadataImportSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>